src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/SystemService.java

```diff
@@ -139,4 +153,23 @@ private String getExceptionMessage(String name, String url, String message) {
         sb.append(" ").append(message);
         return sb.toString();
     }
+
+    /**
+     * Generate internal auth cert CSR
+     * @param distinguishedName
+     * @return
+     * @throws InvalidDistinguishedNameException
```

Comment:
I think it would be good to add a short description about the situation(s) when this exception can occur.

Reply:
Added javadoc description here for the `InvalidDistinguishedNameException`